public/src/js/widgets/columns/fronts-config.html

```diff
@@ -232,6 +232,11 @@
                 <span class="cnf-form__value" data-bind="text: meta.groups"></span>
             <!-- /ko -->
 
+            <div data-bind="visible: $root.priority == 'email' ">
+            <label for="backfillLimit">Maximum number of items to render</label>
+            <input id="backfillLimit" type="number" data-bind="value: meta.displayHints.maxItemsToDisplay">
```

Comment:
Also you may need to add a styling rule for inputs of type "number": https://github.com/guardian/facia-tool/blob/master/public/src/css/style.css#L1157-L1186